pkgs/development/python-modules/retworkx/default.nix

```
@@ -1,62 +1,65 @@
 { lib
-, buildPythonPackage
-, pythonOlder
-, pythonAtLeast
-, python
+, rustPlatform
+, python3
```

inside of a python package set, python will point to the correct interpreter version. python3 is likely to only be aliased to python38.
Causing issues for the python37 version
```suggestion
, python
```